templates/powershell.txt

```diff
@@ -85,7 +85,7 @@ if ($__zoxide_hooked -ne 1) {
     $__zoxide_hooked = 1
```

Comment:
Shouldn't `__zoxide_hooked` also be marked as `global`?

Reply:
Ok, I mark it also as global. Currently, in my profile, I use str.replace: `(zoxide init powershell | Out-String).Replace("function __","function global:__")`. And it works, so I am not sure what is the impact of this variable